module-system/sov-state/src/codec/borsh_codec.rs

```diff
@@ -7,7 +10,7 @@ pub struct BorshCodec;
 
 impl<K> StateKeyCodec<K> for BorshCodec
 where
-    K: borsh::BorshSerialize + borsh::BorshDeserialize,
+    K: BorshSerialize + borsh::BorshDeserialize,
```

Comment:
Nit: could we stick to one approach for both?

Reply:
Done!